Framework/Algorithms/test/PolarizationCorrections/HeliumAnalyserEfficiencyTest.h

```diff
@@ -42,7 +42,7 @@ class HeliumAnalyserEfficiencyTest : public CxxTest::TestSuite {
     MatrixWorkspace_sptr ws1 = generateWorkspace("ws1", x, y);
     auto heliumAnalyserEfficiency = AlgorithmManager::Instance().create("HeliumAnalyserEfficiency");
     heliumAnalyserEfficiency->initialize();
-    heliumAnalyserEfficiency->setProperty("InputWorkspace", ws1->getName());
+    TS_ASSERT_THROWS(heliumAnalyserEfficiency->setProperty("InputWorkspace", ws1), std::invalid_argument &);
     heliumAnalyserEfficiency->setProperty("OutputWorkspace", "P");
```

Comment:
Can we remove the two lines after this, as I'm not sure they will be reached if the test has already thrown by this point?

Reply:
Looking at this again, I think the two lines after this assert are actually reached. I think it's probably sufficient for the purpose of the test just to have the first assert in there, but I don't think it's an issue to keep the extra lines too if preferred, so feel free to go with what you think on this.